src/index.html

```diff
@@ -13,6 +13,112 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+      <div class="stars__star">
+        <a href="#"></a>
+      </div>
+    </div>
```

Comment:
Add empty lines between sibling blocks of HTML for better readability. For example, after the closing </div> of each .stars block.